app-route.html

```diff
@@ -325,7 +327,15 @@
           this._dataInUrl[key] = namedMatches[key];
         }
 
-        this.__setMulti(propertyUpdates);
+        if (this.setProperties) {
+          if (!this.active) {
```

Comment:
`__setMulti` does some black magic that can change readOnly properties unlike `setProperties`